### PR TITLE
More clarification

### PR DESCRIPTION
**Pull request by joinimran**
There was confusion if the license is not assigned to the user.  Updated the information.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9161

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/microsoft-defender-atp/mac-support-license.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Contact your administrator for help.
 
 **Cause:** 
 
-You deployed and/or installed the Microsoft Defender for Endpoint for macOS package ("Download installation package") but you might have run the configuration script ("Download onboarding package").
+You deployed and/or installed the Microsoft Defender for Endpoint for macOS package ("Download installation package") but you might have run the configuration script ("Download onboarding package"). Or You have not assigned the license to the user.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/mac-support-license.md:50]: Looks like there should be some form of continuation of the existing sentence, if the added text should start with "or". I will add an improvement suggestion, but I suspect that @JohanFreelancer9 may have a better suggestion.
```suggestion
You deployed and/or installed the Microsoft Defender for Endpoint for macOS package ("Download installation package"), but you might have run the configuration script ("Download onboarding package"), or you have not assigned a license to the user.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Only @illfated's suggestion.

**Review by illfated — APPROVED** [review]
Looks good to me.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by nam31** [comment]
Hi @Dansimp, PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by illfated** [comment]
(The target pages have moved to the MicrosoftDocs/microsoft-365-docs repository and are updated from there.)

**Comment by illfated** [comment]
New PR targeting the new page: https://github.com/MicrosoftDocs/microsoft-365-docs/pull/4866

**Pull request closed without merging**